types/rebass__forms/rebass__forms-tests.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+import { Label, Input, Select, Textarea, Radio, Checkbox } from '@rebass/forms';
```

Comment:
Can the `TextArea` be added below? Otherwise it is not covered by any test

Reply:
Thank you for your response! I just added `Textarea` component below.